fontforge/encoding.c

```diff
@@ -2101,14 +2105,22 @@ return( NULL );
 
     map = chunkalloc(sizeof(EncMap));
     map->enccount = map->encmax = base + extras;
-    map->map = malloc(map->enccount*sizeof(int32));
-    memcpy(map->map,encoded,base*sizeof(int32));
-    memcpy(map->map+base,unencoded,extras*sizeof(int32));
+    if ( map->enccount>0 ) {
+	map->map = malloc(map->enccount*sizeof(int32));
+	if ( base>0 )
+	    memcpy(map->map,encoded,base*sizeof(int32));
+	if ( extras>0 )
+	    memcpy(map->map+base,unencoded,extras*sizeof(int32));
+    } else
+	map->map = NULL;
```

Comment:
This should be unnecessary, given it's allocated with chunkalloc

Reply:
I suppose, but it seems like skipping these often results in a compiler warning.

Reply:
I dont't think the compiler would warn in this case, it's more likely to warn on uninitialised stack variables.

Reply:
You approved, but all things considered would you prefer I remove these?

Reply:
Probably, but it's not something I'll lose sleep over